src/main/java/jenkins/plugins/slack/pipeline/SlackUploadFileRunner.java

```diff
@@ -24,7 +26,7 @@
 import org.apache.http.util.EntityUtils;
 import org.json.JSONObject;
 
-class SlackUploadFileRunner extends MasterToSlaveCallable<Boolean, Throwable> implements Serializable {
+public class SlackUploadFileRunner extends MasterToSlaveCallable<Boolean, Throwable> implements Serializable {
```

Comment:
Let's keep it out of the public API for now

```suggestion
@Restricted(NoExternalUse.class)
public class SlackUploadFileRunner extends MasterToSlaveCallable<Boolean, Throwable> implements Serializable {
```